OpenRA.Mods.Common/Traits/IdleWander.cs

```diff
@@ -0,0 +1,38 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Will Move to a random location within MoveRadius when idle.",
```

Comment:
I agree that the broken by design trait https://github.com/OpenRA/OpenRA/pull/7222/files#diff-e2c08dd74a071ebf738fa2aaa9ba81dcR86 is terrible. This might be an opportunity to fix it.